### PR TITLE
Updating EC2 GPU section to reflect new instance types

### PR DESCRIPTION
**Pull request by gleeds**
Updating GPU EC2 section to add P2 series of instances launched last month, and adding tip about  spot instances.

### CODE REVIEW

**Comment by jlevy** [comment]
Thanks! LGTM. I'll see if any other GPU experts might want to review too.

File: README.md [hunk 1]
```diff
@@ -699,7 +699,12 @@ EC2
...
 	-	Avoid reusing the original boot keys except by administrators when creating new instances.
 	-	How to avoid sharing keys; how to add individual ssh keys for individual users.
--	**GPU support:** You can rent GPU-enabled instances on EC2. There are [two instance types](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using_cluster_computing.html). Both sport an NVIDIA card (K520, 1536 CUDA cores and M2050, 448 CUDA cores).
+-	**GPU support:** You can rent GPU-enabled instances on EC2 for use in machine learning or graphics rendering workloads.
+  - There are [three generations](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using_cluster_computing.html) of GPU-enabled instances available:
+		- Third generation P2 series offers NVIDIA K80 GPUs in 1, 8 and 16 GPU configurations targeting machine learning and scientific workloads.
+		- Second generation G2 series offers NVIDIA K520 GPUs in 1 or 4 GPU configurations targeting graphics and video encoding.
+		- First generation CG1 instances are still available in some regions in a single configuration with a NVIDIA M2050 GPU.
+  - 🔹Typical GPU workloads are often a good fit for [Spot Instances](#ec2-cost-management).
```
Comment (forwidur) [inline README.md]: I: I think this is pretty task-dependent and maybe this line is better to be removed.

**Comment by forwidur** [comment]
LGTM, outside of a very minor comment, that everyone is free to ignore.

File: README.md (hunk 1 above)
Comment (jlevy) [inline README.md]: No harm in mentioning, but it's also fair that it's not specific to GPU. How about: "As with any expensive EC2 instance types, [Spot instances can offer significant savings](#ec2-cost-management) with GPU workloads when interruptions are tolerable."

**Comment by jlevy** [comment]
👍 Awesome. Thanks @gleeds @forwidur !

**Pull request merged**